src/components/layout/Footer/Footer.js

```diff
@@ -92,16 +92,22 @@ const Footer = ({ children }) => (
                 </li>
               </ul>
             </div>
-            <img src='./images/cards.png' alt='Supported credit cards' />
+            <img
+              className='col text-center'
+              src='./images/cards.png'
+              alt='Supported credit cards'
+            />
           </div>
         </div>
       </div>
     </div>
     <div className={styles.bottomBar}>
       <div className='container'>
-        <div className='row align-items-center'>
-          <div className='col'></div>
-          <div className={'col text-center ' + styles.copyright}>
+        <div className='row'>
+          <div className={'col text-center '}> </div>
+        </div>
+        <div className='row justify-content-center'>
+          <div className={'col text-xs-left ' + styles.copyright}>
             <p>©Copyright 2016 Bazar | All Rights Reserved</p>
```

Comment:
Napis  `Copyright...  ` w wersji desktopowej jest na środku, dopiero po przejściu do szerokości tabletu i poniżej równa się do lewej. Mi to ogólnie nie przeszkadza, niech pozostali zadecydują. Do tego poniżej szerokości 767px zaczyna ten napis przechodzić do kolejnej linii. Fajnie jakby przy tym przejściu ` All Rights Reserved ` w całości przechodziło do kolejnej linii

Reply:
Poniżej 767px przechodzi do kolejnej linii, tylko poniżej 320px nie. To też poprawić?

Reply:
Generalnie od 767 w dół fajnie by było jakby się to podzieliło na 2 części i było pod sobą.